backend/utils/authUtils.js

```diff
@@ -25,6 +35,7 @@ const cognito = new AWS.CognitoIdentityServiceProvider({
 
 exports.signUserUp = (email, password, username) => {
     // Signs the user up and sends a confirmation code to the provided email
+    // Returns a user uuid
```

Comment:
Codacy has a fix for the issue: [Replace `····` with `··`](https://app.codacy.com/gh/cpen321project/cpen321project/pullRequest?prid=9852555)
```suggestion
  // Returns a user uuid
```